integration/mochaTests/README.md

```diff
@@ -0,0 +1,131 @@
+# Mocha Tests
+
+Currently, integration tests are being transitioned over from KITE to Mocha. `mochaTests` contains the tests that are being transitioned from KITE to Mocha.
+
+## Test Types
+
+There are two types of Mocha tests: integration and browser compatibility.
+
+### Integration tests
+Integration tests are minimal tests that run on latest Chrome on MacOs. These tests are used to test the basic functionality across the more popular browsers.
+
+### Browser Compatibility Tests
+Browser compatibility tests are used to test a feature across several different browser and OS combinations. This test is used to make sure that JS SDK features are compatible with a wide variety of browser and OS. There is a default set of browser and OS that are used for this test. In a case where a feature is not compatible with one of the browser or OS then the user can pass in a custom config.
+
+In the following section, we will go over the schema of the custom JSON config.
+#### JSON Config Schema
+
+```json
+{
+   "clients": [
+     {
+        /* 
+          Browser name
+          @required
+        */ 
+        "browserName": "chrome",
+        /* 
+          Browser Version
+          @required
+        */
+        "browserVersion": "latest",
+        /* 
+          Platform
+          @required        
+        */
+        "platform": "MAC"
+     }
+   ]
+}
+```
+## Running tests
+
+### Running Integration Test Locally
+You can run the integration tests locally. By default, integration tests will run on latest version of Chrome that is installed on your machine.
```

Comment:
nit:
- `latest` => `the latest`
- `Chrome that is installed on your machine.` => `Chrome installed on your machine.`